swagger/en/schemas/gameModeStats.yml

```diff
@@ -74,6 +74,10 @@ properties:
     type: number
     description: "[PC only] Number of rank points the player was awarded. This value will be 0 when roundsPlayed < 10"
     minimum: 0
+  rankPointsTitle:
+    type: string
+    description: "[PC Only, Available Dec 19, 2018] Rank title in the form title-level"
+    minimum: "0-0"

```

Comment:
I don't think this is the intended use of the minimum attribute. Does this even work? I'm pretty sure it has to be a number.